src/icons/svgs/controls/icon-moon.svg

```diff
@@ -0,0 +1,10 @@
+<svg width="26px" height="26px" viewBox="0 0 26 26" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+    <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
+        <g transform="translate(-777.000000, -1903.000000)">
+            <g transform="translate(778.000000, 1904.000000)">
```

Comment:
Want to collapse these groups? :D 
should be something like
```suggestion
    <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd" transform="translate(1, 1)">
```

Reply:
it looks like we run these through svgo as part of the build process?

Reply:
ahh did not know that! :)